src/interfaces/toMarcRecords.js

```diff
@@ -13,45 +13,37 @@ export default async function (amqpOperator) {
 		logger.log('info', 'Starting to transform stream to records');
 		let recordNumber = 0;
 		const promises = [];
-		const reader = chooseAndInitReader(contentType);
 
 		// Purge queue before importing records in
 		await amqpOperator.checkQueue(correlationId, 'messages', true);
+		logger.log('info', 'Reading stream to records');
+		const reader = chooseAndInitReader(contentType);
```

Comment:
Declare variables in the block where they are used. This variable is not needed outside the `new Promise` callback.